LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -94,6 +100,42 @@ namespace litecore {
             Warn("SQLite error (code %d): %s", errCode, msg);
         }
     }
+    
+    static void findBlobs(const Array* array, unordered_set<string> &digests, SharedKeys* sharedKeys) {
```

Comment:
There's already an implementation of this in IncomingRev.cc — if it's being done in two places we should move it into a separate file so they can both call it.